src/pages/settings/components/sidebar/styles.tsx

```diff
@@ -1,41 +1,85 @@
 import styled, { css } from 'styled-components'
 
-type Props = {
-  isActive: boolean
-}
+export const SettingsPageSideBarAside = styled.aside`
+  overflow: hidden;
+  position: sticky;
+  padding: 10px 0px;
+  height: 100vh;
+  width: 100%;
+  top: 0;
 
-const SettingsPageSideBarAside = styled.aside`
-  padding: 10px 5px;
+  & > h2 {
+    margin-bottom: 30px;
+    padding-left: 5px;
+    white-space: nowrap;
+  }
 `
 
-const SettingsPageSideBarLi = styled.li<Props>`
-  display: flex;
-  align-items: center;
+export const SettingsPageSideBarLi = styled.li<{ isActive: boolean }>`
```

Comment:
`SettingsPageSideBarLi` is bad naming. Tag shouldn't be as a part of component name